tox.ini

```diff
@@ -5,6 +5,11 @@
 no_package = True
 skip_missing_interpreters = True
 env_list = format, lint, unit
+labels =
+    # Don't run this group in parallel, or with --keep-models because it creates a lot of local VMs.
+    integration = j2-{charm,tls,ha,h-scaling,client}-integration
+    j3-integrat6ion = j3-{charm,tls,ha,h-scaling,client}-integration
+min_version=4
```

Comment:
ah, that seems like weird behaviour. I assumed this would just cause tox to fail if it wasn't the correct version, I'll remove this line in that case.